src/app/navigation/pagination.component.ts

```diff
@@ -113,24 +114,37 @@ export class PaginationComponent {
      *          select option which holds current value selected.
      */
     updateLimit(option) {
-        // when selecting a limit greater than the total the following exception is thrown
-        // throws browser_adapter.js:76 EXCEPTION: Attempt to use a dehydrated detector: PaginationComponent_1 -> ngModelChange
+        // occasionally throws browser_adapter.js:76 EXCEPTION: Attempt to use a dehydrated detector: PaginationComponent_1 -> ngModelChange
```

Comment:
When is 'occasionally'? I am trying to reproduce it now but do not seem to be able to.